crates/common/consensus/src/deneb/beacon_state.rs

```diff
@@ -1790,6 +1790,99 @@ impl BeaconState {
         }
         Ok(())
     }
+
+    // START: This is for prototyping snarkifying the state transition function
+    // TODO: remove this after we are done with the prototyping
```

Comment:
Actually if we're expecting to remove this later, I'm happy to work from a branch and not having to pollute master?

Reply:
Sure I will close the PR